sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
```

Comment:
It is not easy for people to come here and change setting. Also, we should not allow someone directly changing a library.

It would be better if we add these settings in `app/config.php` (as an array probably?)